BibBuddy/app/src/main/java/de/bibbuddy/ExportBibTex.java

```diff
@@ -181,16 +181,19 @@ private String getBibKey(Book book) {
 
   private String getBibNotesFromBook(Book book, NoteDao noteDao) {
     List<Long> notesList = noteDao.getAllNoteIdsForBook(book.getId());
+    NoteTypeLut noteType = NoteTypeLut.TEXT;
```

Comment:
```suggestion
```

Reply:
If I delete this line, do you have a suggestion about how can I get the text notes?

Reply:
In your old code you don't consider the text notes, because you only check if the NoteTypeLut enum NoteTypeLut.TEXT has the value 1. And this is always true. 

The NoteDao `public String findTextById(Long id)` method  should probably be updated.  Check if the `DatabaseHelper.NOTE_FILE_ID` has the same value as the NoteTypeLut.TEXT.

Reply:
Try this out right at the beginning of your function with your debugger and I hope you understand what I mean. The test variable in the debugger shows `test=true`.
```
    boolean test = NoteTypeLut.TEXT.getId() == 1;
```

Reply:
> In your old code you don't consider the text notes types, because you only check if the NoteTypeLut enum NoteTypeLut.TEXT has the value 1. And this is always true.
> 
I am considering only text values, since the enum constructor is called only with parameter 1 that is a Text.